docs/userGuide/contentAuthoring.md

```diff
@@ -274,6 +276,17 @@ To use the searchbar within a navbar, add the following markup to your file. The
 </li>
 ```
 
+### Keywords
+
+You can also specify custom keywords to include under a heading by including keyword class:

```

Comment:
... to ~~include~~ be included under a heading, by ~~including keyword class~~ tagging the words with the `keyword` class.